i18n/module/i18n_module_en.properties

```diff
@@ -158,3 +158,4 @@ login=Login
 never=Never
 invalid_date=Invalid date
 user_profile=User profile
+open_id_connect_account_notification=This account is using OpenID Connect. Updating user account settings needs to be done at the OpenID Connect provider.
```

Comment:
@cooper-joe check here

Reply:
I suggest this text:
`This account is linked to an OpenID Connect identity. Visit the Open ID Connect provider to manage this account's settings.`